### PR TITLE
Cases query

### PR DESCRIPTION
**Pull request by velrest**

### CODE REVIEW

File: addon/caluma-query/queries/base.js [hunk 1]
```diff
@@ -18,12 +18,14 @@ export default class BaseQuery {
     pageSize = null,
     processNew = async (items) => items,
     processAll = async (items) => items,
+    queryOptions = {},
```
Comment (anehx) [inline addon/caluma-query/queries/base.js:21]: This needs to be documented

File: addon/caluma-query/queries/index.js [hunk 2]
```diff
@@ -1,7 +1,12 @@
 import WorkItemQuery from "./work-item";
+import CaseQuery from "./case";
 
 export function allWorkItems(options) {
   return new WorkItemQuery(options);
 }
 
-export default { allWorkItems };
+export function allCases(options) {
```
Comment (anehx) [inline addon/caluma-query/queries/index.js:8]: Document it...

**Review by anehx — CHANGES_REQUESTED** [review]
Code looks good but documentation is inadequate, please fix that.

File: addon/caluma-query/queries/base.js (hunk 1 above)
Comment (velrest) [reply addon/caluma-query/queries/base.js:21]: Done :+1:

File: addon/caluma-query/queries/index.js (hunk 2 above)
Comment (velrest) [reply addon/caluma-query/queries/index.js:8]: Also done

**Comment by anehx** [comment]
Now we still have failing tests

**Comment by velrest** [comment]
@anehx should be fixed now

**Review by anehx — APPROVED** [review]

**Pull request merged**